src/main/java/Dao/SalesManagerDao.java

```diff
@@ -0,0 +1,72 @@
+package Dao;
+import Models.SalesManager;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import Util.Hibernate;
+public class SalesManagerDao {
+    public SalesManagerDao() {
+    }
+    public void createSalesManager(SalesManager salesManager) {
+        Session session = Hibernate.getSessionFactory().openSession();
+        Transaction transaction = null;
+        try {
+            transaction = session.beginTransaction();
+            session.save(salesManager);
+            transaction.commit();
+        } catch (Exception ex) {
+            System.out.println(ex.getMessage());
+            if (transaction != null) {
+                transaction.rollback();
+            }
+        }
+        session.close();
+    }
+    public SalesManager getSalesManager(Long salesManagerId) {
+        Session session = Hibernate.getSessionFactory().openSession();
+        try {
+            SalesManager salesManager = session.find(SalesManager.class, salesManagerId);
```

Comment:
You can use here SM instead of salesManager as a variable, because you are using this variable as local inside the method.